js/src/components/modals/SemRushUpsellAlert.js

```diff
@@ -0,0 +1,42 @@
+/* External dependencies */
+import React from "react";
+import { __, sprintf } from "@wordpress/i18n";
+
+/* Internal dependencies */
+import { Alert } from "@yoast/components";
+import { makeOutboundLink } from "@yoast/helpers";
+
+const PremiumLandingPageLink = makeOutboundLink();
+
+/**
+ * Creates the content for the Yoast SEO Premium upsell alert in SEMrush modal
+ *
+ * @returns {React.Element} The Yoast SEO Premium upsell alert
+ */
+const SemRushUpsellAlert = () => {
+
+	return (
+		<Alert type="info">
+			{
+				sprintf(
+					/* translators: %s : Expands to "Yoast SEO". */
+					__( "Would you like to be able to add these related keyphrases to the %s analysis so you can optimize your content even further?", "wordpress-seo" ),
+					"Yoast SEO"
+				)
+				+ " "
+			}
+			<PremiumLandingPageLink
+				href="https://yoa.st/413">
+				{
+					sprintf(
+						/* translators: %s : Expands to "Yoast SEO Premium". */
+						__( "Check out %s!", "wordpress-seo"),
```

Comment:
Running `grunt eslint` locally, it tells me:
`33:43  error  There must be a space inside this paren`